### PR TITLE
Suggest using config file with `.rb` extension

### LINKED ISSUE #159: Idea: Allow the configuration to be named `.active_record_doctor.rb` (`.rb` suffix) to assist editors
Many editors use the filename suffix to decide how to highlight syntax and load other language hooks.

The name `.active_record_doctor` leaves this as ambiguous to an editor, so it may not automatically select Ruby syntax highlighting.

There are relatively easy workarounds to this problem by either configuring one's editor to special case `.active_record_doctor` or selecting the editor's syntax highlighting manually, but renaming the file would be more universal and friendly to new users.

Other Ruby lint tools, such as rubocop, would also pick up the suffix automatically.

### PR DESCRIPTION
**Pull request by fatkodima**
Fixes #159.

Backwards compatibility is preserved to also accept `.active_record_doctor` files by default (without extension), if present.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ ActiveRecordDoctor::Rake::Task.new do |task|
   task.deps = []
 
   # A path to your active_record_doctor configuration file.
-  task.config_path = ::Rails.root.join(".active_record_doctor")
+  task.config_path = ::Rails.root.join(".active_record_doctor.rb")
```
Comment (jdufresne) [inline README.md:58]: This section it titled "Additional Installation Steps for non-Rails Projects" but this is using a Rails specific function.

Comment (gregnavis) [reply README.md:58]: Great catch! This needs to be fixed.

**Review by gregnavis — CHANGES_REQUESTED** [review]
@fatkodima, I wouldn't like to support two different configuration files. Let's also show a warning when the old file is used, so that users are aware we are migrating to `.active_record_doctor.rb`.

I suggest the following wording:

```
WARNING: configuration was loaded from .active_record_doctor, but it's support will be dropped in favor of .active_record_doctor.rb. Please rename the configuration file to avoid errors in the future.
```

**Comment by jdufresne** [comment]
@gregnavis Do you feel #162 solves what you're looking for? Feedback is welcome.

**Comment by gregnavis** [comment]
Closing in favor of #166.

File: README.md (hunk 1 above)
Comment (gregnavis) [reply README.md:58]: I opened an issue to address that. See #165.

**Pull request closed without merging**